builtin/k8s/platform.go

```diff
@@ -641,6 +648,74 @@ func (p *Platform) Destroy(
 	return rm.DestroyAll(ctx, log, sg, ui)
 }
 
+func (p *Platform) Status(
+	ctx context.Context,
+	log hclog.Logger,
+	deployment *Deployment,
+	ui terminal.UI,
+) (*sdk.StatusReport, error) {
+	sg := ui.StepGroup()
+	defer sg.Wait()
+
+	step := sg.Add("Gathering health report for Kubernetes platform...")
+	defer step.Abort()
```

Comment:
I recommend using `defer func() { step.Abort() }()` in case we ever add other steps.